app/models/qa/iri_template/variable_map.rb

```diff
@@ -1,39 +1,42 @@
 # Provide access to iri template variable map configuration.
 # See https://www.hydra-cg.com/spec/latest/core/#templated-links for information on IRI Templated Links - Variable Mapping.
 # TODO: It would be good to find a more complete resource describing templated links.
+require 'erb'
+
 module Qa
   module IriTemplate
     class VariableMap
+      include ERB::Util
+
       TYPE = "IriTemplateMapping".freeze
       attr_reader :variable
-      attr_reader :default
+      attr_reader :default, :required, :encode, :property
```

Comment:
I do not think the required attr_reader is used anywhere.  Since it is private it might be better to just remove it.  It looks like you utilize required? which uses @required